app/assets/javascripts/darkswarm/controllers/checkout/accordion_controller.js.coffee

```diff
@@ -11,16 +11,14 @@ Darkswarm.controller "AccordionCtrl", ($scope, localStorageService, $timeout, $d
     $scope.accordion[section] = true
     # If we call scrollTo() directly after show(), when one of the accordions above the
     # scroll location is closed by show(), scrollTo() will scroll to the old location of
-    # the element. Putting this in a 100 ms timeout is enough delay for the DOM to
+    # the element. Putting this in a 50 ms timeout is enough delay for the DOM to
     # have updated.
     $timeout (->
-      console.log("Scrolling to ##{section}")
       $document.scrollTo $("##{section}"), offset_height, 500), 50
```

Comment:
You could make it a bit clearer like this:
```coffee
  millis_until_dom_updated = 50
  $timeout (->
    $document.scrollTo $("##{section}"), offset_height, 500),
    millis_until_dom_updated
```

Then you almost don't need the comment and people don't have to remember the function's parameter signature to know where the `50` belongs to.

You derived that value by experimentation, right? Does it depend on the browser speed updating the dom or is there a timeout in the accordion code? If we knew that the accordion code updates in an animation that takes 50 ms or so, then we would know exactly how long to wait.